tf-modules/ingest/main.tf

```diff
@@ -3,3 +3,7 @@ terraform {
     aws = ">= 2.31.0"
   }
 }
+
+locals { 
+    all_non_system_buckets = [for k, v in var.buckets : v.name if v.type != "internal"]
```

Comment:
`all_non_internal_buckets` feels a bit more clear, since this name relies on the knowledge that internal buckets = system buckets

Reply:
57b92d1bd